spec/models/video_spec.rb

```diff
@@ -1,5 +1,40 @@
 require 'spec_helper'
 
 describe Video do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  it {should belong_to(:category)}
+  it {should validate_presence_of(:title)}
+  it {should validate_presence_of(:description)}
+
+
+  # km01: replaced with shoulda
+  # it "a video can save itself" do
+  #   video = Video.new(title:"Kingpin", description:"Best hairdoos in bowling", url_small_cover:"smallcover.jpg",
+  #                     url_large_cover:"largecover.jpg")
```

Comment:
Because the line needs to split up I would do it on each comma for consistency:

```
video = Video.new(
  title: "Kingpin",
  description: "Best hairdoos in bowling",
  url_small_cover:"smallcover.jpg",
  url_large_cover:"largecover.jpg"
)
```

Also note the space after the `:`